decentraland/unity-explorer
# fix: analytics integration cloud ci build

## What does this PR change?

- moved logic for the write key from old build script to new cloud build script
- fix write key being not serialized in SO
- improved fallback to not break experience
- removed unused old BuildScript

## How to test the changes?

1. Launch the explorer
2. Check it is running normally

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
